<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>recur_expenses_mst</class>
 <widget class="QDialog" name="recur_expenses_mst">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>840</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Recurring Expenses Listing</string>
  </property>
  <widget class="QGroupBox" name="gb_search">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>821</width>
     <height>191</height>
    </rect>
   </property>
   <property name="title">
    <string>Search Recurring Expenses</string>
   </property>
   <widget class="QPushButton" name="btn_search">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>160</y>
      <width>156</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>21</y>
      <width>671</width>
      <height>126</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="1">
      <widget class="QComboBox" name="lst_cat">
       <item>
        <property name="text">
         <string>&lt;Select one&gt;</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="lst_recurType">
       <item>
        <property name="text">
         <string>&lt;Select one&gt;</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Daily</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Monthly</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Yearly</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="txt_name"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="lbl_name">
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Recurring Expenses Name:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="lbl_cat">
       <property name="text">
        <string>Category:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lbl_amount">
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Recurring Amount From (RM):</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="txt_amout_from"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="lbl_type">
       <property name="text">
        <string>Recurring Type:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lbl_amount_2">
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Recurring Amount To (RM):</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="txt_amout_to"/>
     </item>
    </layout>
   </widget>
   <zorder>layoutWidget</zorder>
   <zorder>btn_search</zorder>
  </widget>
  <widget class="QGroupBox" name="gb_listing">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>821</width>
     <height>341</height>
    </rect>
   </property>
   <property name="title">
    <string>Recurring Expenses Listing</string>
   </property>
   <widget class="QTreeWidget" name="tw_recurList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>801</width>
      <height>281</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string notr="true">1</string>
     </property>
    </column>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>20</y>
      <width>239</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btn_add">
       <property name="text">
        <string>Add New</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_edit">
       <property name="text">
        <string>Edit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_delete">
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>txt_name</tabstop>
  <tabstop>txt_amout_from</tabstop>
  <tabstop>txt_amout_to</tabstop>
  <tabstop>lst_cat</tabstop>
  <tabstop>lst_recurType</tabstop>
  <tabstop>btn_search</tabstop>
  <tabstop>btn_add</tabstop>
  <tabstop>btn_edit</tabstop>
  <tabstop>btn_delete</tabstop>
  <tabstop>tw_recurList</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
